docs/se/config/mutability-support.adoc

```diff
@@ -130,15 +118,15 @@ Config config = Config.create(
         ConfigSources.file("conf/config.properties")
                 .changeWatcher(FileSystemWatcher.create())                         // <2>
                 .optional(),
-        ConfigSources.classpath("application.properties")
+        ConfigSources.file("my.properties")
```

Comment:
Do we not support classpath any more? If so, there is a point above this piece of code that also needs to change i.e. 

> watch(Path watchedPath) - a filesystem-specific strategy to watch specified path. You can use this strategy with the file and classpath built-in config sources.

Reply:
My bad, you said polling is not supported and the above point is for watching so it should be fine.